docs/README.rst

```diff
@@ -26,45 +28,58 @@ Key features:
 - simple, one-time-use codes that are easy to transcribe
 - secure (full-strength keys via SPAKE2)
-- integrate with tools that can listen on or connect to localhost
+- end-to-end encryption (and no possibility for unencrypted application data)
+- integrate with any tools that can listen on or connect to localhost
 
-This allows an author to write a "glue" program in *any language* that ties together unchanged networked progams. The communcation channel is: set up *without pre-shared secrets*; *fully encrypted*; and *survives IP address changes or outages*. All this with *no action required at the application level*, it is just a normal localhost TCP (or UNIX) streaming socket.
+This allows an author to write a "glue" program in *any language* that ties together unchanged networked progams.
+The communcation channel is:
+- set up *without pre-shared secrets*;
+- *fully encrypted*;
+- and *survives IP address changes or outages*.
 
+All this with *no action required at the application level*, it is just a normal localhost TCP (or UNIX) streaming socket.
 
-Motivational Example
---------------------
+
+✍ Motivational Example
+----------------------
 
 When pair-programming using `tty-share <https://tty-share.com/>`_ one handy option is to use the default, public server.
 However, *I don't like the idea of sending keystrokes over a third-party server* that I don't run.
+(Please note: I have **no** reason to believe this particular person is doing anything nefarious!)
+
+I could fire up such a server myself and use it with my friends...
 
-I could fire up such a server myself and use it with my friends, but with ``fowl``, one side can run a localhost ``tty-share`` server and the other side can run a ``tty-share`` client that connects to a ``localhost`` endpoint -- data flows over the wormhole connection (only).
+...but with ``fowl``, one side can run a localhost ``tty-share`` server and the other side can run a ``tty-share`` client that connects to a ``localhost`` endpoint -- data flows over the wormhole connection (only).
 
 Key advantage: *no need to expose keystrokes to a third-party server*.
 
 Additional advantage: *no need to set up a server on a public IP address*.
 
 
-Why is This Particular Yak Being Shorn?
----------------------------------------
+🐃 Why is This Particular Yak Being Shorn?
+------------------------------------------
 
 I wanted to write a pair-programming application in Haskell, but didn't want to implement Dilation in the Magic Wormhole Haskell library (maybe one day!)
 
 It also occurred to me that other people might like to experiment with Magic Wormhole (and advanced features like Dilation) in languages that lack a Magic Wormhole implementation -- that is, most of them!
 
 So, the first step in "write a Haskell pair-programming utility" became "write and release a Python program" :)
 
+(p.s. the next-higher level Yak is now online at `sr.ht <https://git.sr.ht/~meejah/pear-on>`_ but not "released")
```

Comment:
I was actually considering asking you about Sourcehut already last year but deemed it inappropriate in the end, but since I now see that you are using it and are setting up your new projects there then let me mention that if you chose to move this project there as well, I would be very much in favour.

Reply:
Yeah, trying it out for Pear-On -- do you know if there's a migration script etc? (Don't have a lot of PRs or Issues on this repo, but would love to give this a go as a test-migration....)